### ISSUE TITLE
Can not post event with image

### ISSUE THREAD

**Issue by Leonidaz0r** [amiv-eth/amivapi#149]
I tried creating an event with an image using the following script:

```
import requests
import json

data = {
    'title_en': 'Event mit Bild 2',
    'catchphrase_en': 'yey more pics',
    'description_en': 'more picsssss',
    'show_infoscreen': True,
    'priority': 5,
    'time_advertising_start': '2016-01-02T00:00:00Z',
    'time_advertising_end': '2018-01-05T00:00:00Z'
}

files = {
    'img_infoscreen': open('image.png', 'rb')
}

print(requests.post('http://127.0.0.1:5000/events', data=data, files=files,
                    headers={'Authorization': 'root',
                             'Content-Type': 'multipart/form-data'},
                    verify=False).json())
```

This leads to a response 400 empty form data.

1. This does not make sense. The form data can be seen in the request content, but it is not decoded into request.form. The 400 then gets raised here: https://github.com/nicolaiarocci/eve/blob/develop/eve/methods/common.py#L193

2. Is there currently any way to post an event with an image?

**Comment by Leonidaz0r** [comment]
I figured out it works, if I delete the boolean value.

Is there any way to send boolean values in multipart/form-data?

**Comment by Leonidaz0r** [comment]
There is now. This was solved by NotSpecial/eve@42be190b924be5fb50e6b496a7a4832f85d1167a. One can now post booleans as strings and then have everything in one multipart/form-data request.
